metadata.json

```diff
@@ -87,13 +78,6 @@
     },
     {
       "operatingsystem": "Gentoo"
-    },
-    {
-      "operatingsystem": "Fedora",
-      "operatingsystemrelease": [
-        "37",
```

Comment:
Why drop Fedora support instead of updating the supported versions to `["40", "41"]`?